src/amo/api/index.js

```diff
@@ -41,6 +41,25 @@ export function submitReview({
     });
 }
 
+export function getAddonReviews({ addonSlug } = {}) {
+  return new Promise((resolve) => {
+    if (!addonSlug) {
+      throw new Error('addonSlug cannot be falsey');
+    }
+    resolve(callApi({
+      endpoint: `addons/addon/${addonSlug}/reviews`,
+      method: 'GET',
+    }));
+  })
+    .then((response) => {
+      // TODO: implement paging through response.next
+      if (response.next) {
+        throw new Error('paging is not yet implemented');
```

Comment:
Yes it would because otherwise the additional reviews would be hidden. I feel like paging should probably be a release blocker but I didn't want to wedge it in here since it could get complex with recursion.